app/styles/_base.scss

```diff
@@ -0,0 +1,86 @@
+// Border box all the things by default
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+html {
+  background-color: $html-background-color;
+  height: 100%;
+}
+
+body {
+  color: $text-color;
+  @include font();
+  font-size: $base-font;
+  padding-bottom: 20px;
+}
+
+noscript {
+  color: $error-text-color;
+  display: block;
+  margin: 160px 0 0 0;
+}
+
+a {
+  color: $link-color;
+  cursor: pointer; // Use the correct cursor for anchors without an href
+
+  &:active {
+    outline: none;
+  }
+
+  &:focus {
+    outline: 1px dotted $link-color;
+  }
+}
+
+header {
+  h1,
+  h2,
+  h3 {
+    @include headerFont();
```

Comment:
> _app/styles/_base.scss:43_ [W] NameFormat: Name of mixin `headerFont` should be written in all lowercase letters with hyphens instead of underscores

Meh. Not sure if you really want to follow all the **scss-lint** suggestions, or if we maybe want to disable some of them in the aoo/styles/.scss-lint.yml file.